main.go

```diff
@@ -134,9 +144,23 @@ func main() {
 		}
 	}
 
+	setupLog.Info("Create a composer client")
+	httpClient, err := createClient()
```

Comment:
Sorry for raising it only now, but if we already have a method that creates a client, why not have it return the `composerClient`